example-adapter/src/main/java/org/niis/xrd4j/exampleadapter/ExampleAdapter.java

```diff
@@ -57,6 +61,7 @@
     private String namespaceSerialize;
```

Comment:
You should add your name as an author (with the `@author` annotation) in the beginning of the class.

Reply:
Added